src/content/release-notes.mdx

```diff
@@ -47,6 +47,17 @@ This version is compatible with Kubernetes versions 1.26 to 1.29
 - Support for Kubernetes [1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md) has been removed in this release.
 
 
+## 1.19.4
+
+28 May 2024
+
+This version is compatible with Kubernetes versions 1.25 to 1.29
+
+### Improvements
+
+- Updated CLI version in the backend to 2.26.2  <!-- 8137 -->
```

Comment:
Same comments than in https://github.com/okteto/docs/pull/746/files#r1616966429. Let's try to add more context about why we increased the CLI, so customers might know if it is relevant for them or not